web/styles/compose.css

```diff
@@ -338,7 +338,8 @@
         }
     }
 
-    &.warning {
+    &.warning,
+    &.warning-style {
```

Comment:
Why do we need to add a `warning-style` rather than using the existing `warning`?

Reply:
The banners with `warning` class have a different behavior, but here we only needed to copy their styling, not functionality. Hence I created this new class which only affects the styling.